### ISSUE TITLE
request to add store profile on disk

### ISSUE THREAD

**Issue by Advantaged** [ckb-next/ckb-next#757]
## What is the program's current behaviour?
Some devices don't allow storing profiles on KB-hardware "Save to Hardware"
or profile cannot be loaded from disk "Load Profile from Disk".

## What would you like it to do instead?
Right mouse-click on profile-name, context-menu opening (as already do), add context-menu-option e.g. "Save profile to disk", this open file-manager for navigation and store profile on the disk.
For "Load Profile from Disk" add opening context-menu by 'right-click' on profile-list-empty-space, add "Load Profile-s from Disk", this open file-manager for navigating to stored profile and load it.

## Any other notes & comments?
Basically just add 'open context-menu by 'right-click' on profile-list-empty-space' and add there two submenus "Load Profile-s from Disk" and "Save profile to disk" with file-manager-browsing  to load and store relatives profile.

Some people send many time to configure one or more profiles, in this way they don't need to spend the time by each install of linux-distro.
